ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -219,8 +220,15 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
       homeOwnerGroups = [...getMhrRegistrationHomeOwnerGroups.value]
     }
     // For Mhr Transfers with Removed Groups, assign a sequential groupId
-    const transferDefaultId = homeOwnerGroups.find(group => group.action !== ActionTypes.REMOVED)?.groupId ||
+    // If WILL flow, add new executor to existing group instead of incrementing the group
+    let transferDefaultId = null
```

Comment:
Setting `transferDefaultId = groupId` and removing the `else` clause will have the same effect.